media-gfx/inkscape/inkscape-1.1.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
```

Comment:
Do you really need an interval for only 2 python versions here?

Reply:
No, I don't need it. This used more often than `{8,9}` in `::gentoo`.

Reply:
Anyway, do you want me to change it?

Reply:
Changed to `{8,9}` in recent FP.